torch/distributed/pipelining/PipelineStage.py

```diff
@@ -267,15 +272,16 @@ def get_fwd_recv_ops(self) -> List[dist.P2POp]:
         Returns a list of ops that are needed to receive the input arguments
         for this stage.
         """
-        recv_infos: Tuple[InputInfo, ...] = self.args_recv_info[self.fwd_chunk_id]
```

Comment:
nit: might also be good to assert the chunk_id is in the dict (for both fwd/bwd).  just so we can issue a nicer message then keyerror, and mention that the schedule is probably wrong or something like that

Reply:
good idea, will follow up in a PR  + test adding this